### DISCUSSION TITLE
Problem to obtain publishing rights for custom apps on Managed Play Store

### DISCUSSION THREAD

**Discussion by SilkSama** [fastlane/fastlane#17824] [General]
Hi, 
I try to use` fastlane run get_managed_play_store_publishing_rights` to obtain rights to publish a private app but when I open the link with the account's owner of a managed play store associated to an entreprise created with the Android Management API, I have an error.
The error says that only **the owner of the Play console can deploy private application for my organisation**. and I do not understand why.. 
Thank you in advance for your help

**Comment by rogerluan** [comment]
Hey @SilkSama 👋 

I'm not sure if that issue is related to _fastlane_. Are you able to do that outside of fastlane, manually, by any chance?
